### PR TITLE
Update Ukrainian and Russian translations

### PR DESCRIPTION
**Pull request by Minoru**
I'll merge this once someone—anyone—took a look and didn't find any obvious flaws.

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -712,7 +712,6 @@ msgid "Browser returned error code %i"
 msgstr "Браузер повернув код помилки %i"
 
 #: src/feedlistformaction.cpp:299 src/itemlistformaction.cpp:487
-#, fuzzy
 msgid "Do you really want to mark this feed as read (y:Yes n:No)? "
 msgstr "Ви впевнені, що хочете відмітити усі теми як прочитані (y:Так n:Ні)? "
```
Comment (der-lyse) [inline po/uk.po:716]: I still don't know any Ukrainian, but is this translation correct? The Russian translation got updated and in the German version there was a slight difference (all feeds vs. this feed) as well.

Comment (Minoru) [reply po/uk.po:716]: Oh right, the translation says "all feeds" rather than "this feed". Thanks, will fix in a second!

Comment (der-lyse) [reply po/uk.po]: Probably good now. ;-)

**Review by der-lyse — APPROVED** [review]

**Comment by Minoru** [comment]
Thanks for taking a look! While editing, I actually spotted another typo (repeated "не" twice), and then decided I want to translate one of the messages differently. So here's one more fixup. Hopefully the last one :)

**Review by der-lyse — APPROVED** [review]
Third time lucky, probably alright now. ;-) Didn't see anything obviously wrong.

The only thing I now noticed is that in line 765 a backtick and single quote are used, in 1224 on the other hand it's surrounded by single quotes. But the English original uses the same inconsistency. So, yeah, that's that. Something for another time to fix I reckon.

**Comment by der-lyse** [comment]
I can add the inconsistent use of different quotes to #1525 if you like.

**Comment by Minoru** [comment]
Quotes inconsistency is tracked separately, in #901, because I should make some research and make a decision before I can change anything. Or, well, maybe someone else will :)

Okay, let's merge this already. Thanks for being so patient!

**Comment by der-lyse** [comment]
Ah, of course there's a different story for that already. :-D Thanks!

**Pull request merged**